### PR TITLE
added scripts/pandas_spike.py

### PR DESCRIPTION
**Pull request by CyrusDobbs**
### Link to ticket
[#PAB-4](https://trello.com/c/VGesc44F)

### Change description
Adding a script documenting basic pandas commands for loading in an excel file.

- [x] Unit tests and other appropriate tests added or updated
- [x] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
N/A

### CODE REVIEW

File: scripts/pandas_spike.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import pandas as pd
```
Comment (gidsg) [inline scripts/pandas_spike.py:1]: Can we add this dependency to the requirements?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
 ### Change description
 _A brief description of the pull request_
 
-- [] Unit tests and other appropriate tests added or updated
-- [] README and other documentation has been updated / added (if needed)
-- [] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")
+- [ ] Unit tests and other appropriate tests added or updated
+- [ ] README and other documentation has been updated / added (if needed)
+- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")
```
Comment (gidsg) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Can we revert these changes?

Comment (gidsg) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: the reason for no blank spaces, is so github renders a checkbox and adding an `X` in between the brackets will check it.

File: scripts/pandas_spike.py (hunk 1 above)
Comment (CyrusDobbs) [reply scripts/pandas_spike.py:1]: there is no requirements.txt in the repo yet. want me to create one?

File: scripts/pandas_spike.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+path = r"data/DLUCH_Data_Model_V3.4.xlsx"
```
[the remark this answers is not in the document]
Comment (CyrusDobbs) [reply scripts/pandas_spike.py]: Added this

**Comment by CyrusDobbs** [comment]
[SPIKE Investigate tooling for ingesting excel spreadsheets](https://trello.com/c/VGesc44F/4-spike-investigate-tooling-for-ingesting-excel-spreadsheets)

File: scripts/pandas_spike.py (hunk 1 above)
Comment (gidsg) [reply scripts/pandas_spike.py:3]: Ah, good poink, think we'll be adding pip compile in a separate PR, so fine to hold off from this for now,

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (gidsg) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: Ah, my bad.

**Review by gidsg — APPROVED** [review]
👍
